README.md

```diff
@@ -106,6 +113,11 @@ echo "deb http://apt.postgresql.org/pub/repos/apt/ ${codename}-pgdg main ${PG-MA
     /etc/apt/sources.list.d/postgresql.list
 ```
 
+```bash
+cd extbuilder
+docker build --tag=citus/extbuilder-{pg-major} .
```

Comment:
same here, would tags on the same image help keeping our repo clean of all kinds of similarly named images?

Reply:
https://github.com/citusdata/the-process/issues/39